build.gradle

```diff
@@ -119,7 +119,13 @@ project('common') {
             project(':common').sourceSets.main.java
             project(':shared:protocol').sourceSets.main.java
         }
+        dependencies {
```

Comment:
You're adding a runtime dependency on Lombok. This has been removed with issue #3289. Please do not add them back.

Reply:
No. This is inside of the 'javadoc' block, so it should not affect the main dependencies above.

Reply:
it's actually Tom's code, it did fixed hundreds of build errors/warnings; and based on this changes the left 18 errors get fixed. looking #3289 ...